views/dnd/5e/srd/monsters.ejs

```diff
@@ -0,0 +1,421 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>D&D | 5e | SRD | Monsters | Far Reach Co.</title>
+  <link rel="stylesheet" href="/style.css" />
+  <link rel="stylesheet" href="/info.css" />
+  <script src="/lib/MobileNavHandler.js" defer></script>
+  <%- include('../../../partials/icons') %>
+</head>
+<%- include('../../../partials/google') %>
+
+<body>
+  <%- include('../../../partials/nav') %>
+  <div class="container-fluid p-3 info-container">
+    <%- include('../../../partials/dnd/srdintro') %>
+    <br />
+    <div class="inline-div">
+      <small>
+        <a href="/dnd/5e/srd/contents">Contents</a> ->
+        <div style="color: var(--green)">Monsters</div>
+      </small>
+    </div>
+    <hr />
+    <h2>Monsters</h2>
+    <ul>
+      <% data.forEach((mon) => { %>
+      <li>
+        <a href="#<%= mon.index %>"><%= mon.name %></a>
+      </li>
+      <% }); %>
+    </ul>
+    <br />
+    <!-- how to make a monster -->
+    <% data.forEach((mon) => { %>
+    <% const spellsPageBase = '/dnd/5e/srd/spells#'; %>
+    <div class="inline-div" id="<%= mon.index %>">
+      <h2><%= mon.name %></h2>
+      <div class="inline-div">
+        <i><%= mon.size %>,</i>
+        <i><%= mon.type %>,</i>
+        <i><%= mon.alignment %></i>
+      </div>
+    </div>
+    <div class=" indent">
+      <!-- armor class -->
+      <div class=" inline-div">
+        <b>Armor Class</b>
+        <div>
+          <% if (mon.armor_class && mon.armor_class.length) { %>
+          <% mon.armor_class.forEach((acEntry, acIndex) => { %>
+          <span><%= acEntry.value %> (<%= acEntry.type %>)</span>
+          <%= acIndex < mon.armor_class.length - 1 ? ', ' : '' %>
+          <% }) %>
+          <% } else { %>
+          —
+          <% } %>
+        </div>
+      </div>
+      <!-- hitpoints------------------- -->
+      <div class="inline-div">
+        <b>Hit Points</b>
+        <div> <%= mon.hit_points%></div>
+        <i>(<%= mon.hit_points_roll%>) </i>
+      </div>
+      <!-- speeds------------------- -->
+      <div class="inline-div">
+        <b>Speed:</b>
+        <div>
+          <% 
+            const speed = mon.speed || {};
+            const order = ['walk','burrow','climb','swim','fly']; 
+            const labels = { walk:'Walk ', burrow:'Burrow ', climb:'Climb ', swim:'Swim ', fly:'Fly ' };
+          %>
+          <% order.forEach(speedType => { 
+            const speedValue = speed[speedType];
+            if (speedValue) { 
+          %>
+          <div>
+            <%= labels[speedType] %><%= speedValue + ',' %><%= (speedType === 'fly' && speed.hover) 
+                  ? ' (hover)' 
+                  : '' %>
```

Comment:
It would be better if stuff like this was wrapped in its own div if the parent has other tags nested in it but it's fine